dashboard/src/pages/cloud/api/token-auto.tsx

```
@@ -0,0 +1,230 @@
+        <h2>Invalid Request</h2>
+        <p>The authentication request is missing required information.</p>
+        <p style={{ fontSize: "14px", color: "#666" }}>Please try again from your application.</p>
+      </div>
+    );
+  }
+
+  // Get user's tenants and ledgers
+  const { data: tenantsData, error: tenantsError } = cloud.getListTenantsLedgersByUser();
+
+  // Create ledger mutation
+  const createLedgerMutation = useMutation({
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)